Source/JavaScriptCore/assembler/AssemblerCommon.h

```diff
@@ -323,6 +323,15 @@ static void* performJITMemcpy(void *dst, const void *src, size_t n);
 template<MachineCodeCopyMode copy>
 ALWAYS_INLINE void* machineCodeCopy(void *dst, const void *src, size_t n)
 {
+    bool isAligned = !(reinterpret_cast<uintptr_t>(dst) & 3);
```

Comment:
Nit, this could be `bool isAligned = dst == roundUpToMultipleOf<4>(dst)`, which might b e a bit clearer.

Reply:
Ah, perfect!